### DISCUSSION TITLE
can not import json file in webpack config file itself

### DISCUSSION THREAD

**Discussion by kevinwang930** [webpack/webpack#14233] [Q&A] [answered]
webpack handle json import automatically in bundle process, but I can not import json file in the config itself. if there is already solution please let me know, thanks !  P.S: I use babel and js

**Comment by alexander-akait** [comment]
How you improve json file and there?

**Comment by kevinwang930** [comment, marked as answer]
Sorry My question seems unrelated to webpack which is just a module bundler.  In node environment, we can require json file directly, in ecmaScript, json module is a stage 3 proposal [json module](https://github.com/tc39/proposal-json-modules) to import json we need other transpiler like ts-node.
